Northstar.CustomServers/mod/cfg/autoexec_ns_server.cfg

```diff
@@ -1,12 +1,13 @@
 ns_server_name "Unnamed Northstar Server" // server name
 ns_server_desc "Default server description" // server description
 ns_server_password "" // server password
+ns_server_tags "" // max 5, separated by spaces
 ns_report_server_to_masterserver 1 // whether this server should report itself to the masterserver
 ns_report_sp_server_to_masterserver 0 // whether this server should report itself to the masterserver if started on a singleplayer map
 
 ns_auth_allow_insecure 0 // keep this to 0 unless you want to allow people to join without masterserver auth/persistence
-ns_erase_auth_info 1 // keep this to 1 unless you're testing and crashing alot, so you don't have to go through the northstar lobby to reauth
-ns_player_auth_port 8081 // this can be whatever, make sure it's portforwarded over tcp
+ns_erase_auth_info 1 // keep this to 1 unless you\'re testing and crashing alot, so you don\'t have to go through the northstar lobby to reauth
+ns_player_auth_port 8081 // this can be whatever, make sure it\'s portforwarded over tcp
 
```

Comment:
I'd recommend changing this to not include the contractions. Also not very relevant to the rest of the PR, but fine.

Reply:
Agree. Just added the escaped characters cause vscode wasn't having a good time.